hw9/deco_file.py

```diff
@@ -0,0 +1,22 @@
+from datetime import datetime
+
+
+def decorator(func):
+    def wrapper(*args, **kwargs):
+        t = datetime.now()
+        print(f'====\nFunction call time - {t.strftime("%H:%M:%S")}\n====')
+        func(*args, **kwargs)
```

Comment:
assign a result of `func` call to some variable and return it after the last print here to be able to decorate functions that return some value

Reply:
Remove this line (line 8) since you return the function call at the line 12